wiki/Legal/ja.md

```diff
@@ -6,3 +6,4 @@
 - [osu! 音楽ライセンス](Music_licensing)
 - [osu! プライバシーポリシー](Privacy)
 - [osu! サービス利用規約](Terms)
+- [SCTL](SCTL)
```

Comment:
should add this to other translations as well

Reply:
feel free to add.